be/src/vec/sink/result_sink.cpp

```diff
@@ -125,7 +125,7 @@ Status VResultSink::close(RuntimeState* state, Status exec_status) {
             state->fragment_instance_id());
 
     VExpr::close(_output_vexpr_ctxs, state);
-    _closed = true;
+    DataSink::close(state, exec_status);
```

Comment:
```suggestion
    return DataSink::close(state, exec_status);
```

Reply:
There are other places which have same problem. Please fix them too.

Reply:
I will update it, it is strange, actually I fix all of them at night of yesterday. It seems that I made some mistakes when format a patch.